src/transaction/recovery.c

```diff
@@ -972,10 +972,17 @@ struct rvfun RV_fun[] = {
 
   {RVHF_MVCC_UPDATE_OVERFLOW,
    "RVHF_MVCC_UPDATE_OVERFLOW",
-   heap_rv_undo_update,
-   heap_rv_redo_update_and_update_chain,
-   log_rv_dump_hexa,
-   log_rv_dump_hexa}
+   heap_rv_nop,
+   heap_rv_nop,
```

Comment:
For redo, I think instead heap_rv_nop we need heap_rv_update_chain_after_mvcc_op.